upgrade/1.2/scripts/upgrade/csm-service-upgrade.sh

```diff
@@ -81,7 +81,7 @@ state_name="POST_CSM_UPGRADE_APPLY_POD_PRIORITY"
 state_recorded=$(is_state_recorded "${state_name}" $(hostname))
 if [[ $state_recorded == "0" ]]; then
     echo "====> ${state_name} ..."
-    /usr/share/doc/csm/upgrade/1.0/scripts/upgrade/add_pod_priority.sh
+    /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/add_pod_priority.sh
```

Comment:
if we did this in 1.0, is it needed here?  I am sure it is idempotent, so just more curious